docs/team/kwanhw.md

```diff
@@ -47,7 +49,5 @@ Given below are my contributions to the project.
         * Added use cases for tags
 
 * **Community**:
-    * `to be added soon`
-
-* **Tools**:
-    * `to be added soon`
+    * Reviewed PRs within the team ([20+ comments](https://nus-cs2103-ay2122s2.github.io/dashboards/contents/tp-comments.html) across [50+ PRs](https://github.com/AY2122S2-CS2103-W17-1/tp/pulls?q=is%3Apr++is%3Aclosed+reviewed-by%3Akwanhw))
```

Comment:
The number here don't look correct.

Reply:
Amended in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/322/commits/9acfd793b792ca9c3e94edf65a30df349281b70d